v1/pipeline/stages/ci/caching.yaml

```diff
@@ -13,6 +13,8 @@ properties:
   key:
     description: Key provides a caching key.
     type: string
+  override:
+    type: boolean
```

Comment:
We need a description for this variable. Can we please check with the product team on what this should be called? Maybe `force_push` is more readable?

Reply:
Override is the same in save cache step. Pranav said keep it the same